bash/lsst/sitcom/2023/package_setup-obsrun-2023-03B.sh

```diff
@@ -0,0 +1,226 @@
+
+## Verify the proper build cycle is used
+LSST_IMAGE='w_2023_09_c0029.004'
+
+# Jira ticket number (used as a branch) to support this run
+TICKET='DM-38260'
+
+printf "This run uses the image of ${LSST_IMAGE}\n"
+
+RVAL=$(echo $JUPYTER_IMAGE_SPEC | cut -d ':'  -f 2)
+if [[ $RVAL == *"${LSST_IMAGE}"* ]]; then
+  printf "Nublado image is correct\n"
+else
+  printf "Nublado image is incorrect. You have ${RVAL} but need ${LSST_IMAGE}. Exiting\n"
+  exit 1
+fi
+
+# Verify the proper jupyter build is being used
+REQUIRED_BUILD='lsst-scipipe-5.1.0'
+if [ ${LSST_CONDA_ENV_NAME} == ${REQUIRED_BUILD} ]; then
+  printf "Conda environment is correct\n"
+else
+  printf "Conda environment is incorrect. You have ${LSST_CONDA_ENV_NAME} but need ${REQUIRED_BUILD}.\n"
+  printf "Are you running the right container? \n"
+  exit 1
+fi
+
+# If running from Nublado instance than only use the following
+setup lsst_distrib
+
+## Verify the proper lsst_distrib is loaded
+LSST_DISTRIB_VER=${LSST_IMAGE:0:9}
+
+RVAL=$(eups list lsst_distrib)
+if [[ $RVAL == *"${LSST_DISTRIB_VER}"* ]]; then
+  printf "Nublado weekly is correct\n"
+else
+  printf "Nublado weekly is incorrect. You have ${RVAL} but need ${LSST_DISTRIB_VER}. Exiting\n"
+  exit 1
+fi
+
+# The packages will be cloned and installed in the following folder
+# It is strongly recommended NOT to modify this
+REPOS=$HOME"/auto-op-env-packages/"
+
+# Check if the directory exists
+if [ -d $REPOS ]; then
+  printf "Directory "$REPOS" exists. Remove directory and contents and re-run.\n"
+  exit 1
+fi
+# Start cloning and setting up packages
+printf 'Repositories will cloned and setup in the directory:'$REPOS"\n"
+mkdir -v ${REPOS}
+
+# Code below assumes that the branch exists already
+# See package_branch_creation.sh if this needs to be done 
+
+printf '\nSetting up lsst-dm/atmospec \n'
+cd $REPOS
+git clone https://github.com/lsst-dm/atmospec.git
+cd atmospec
+setup -j -r .
+git fetch --all
+git checkout tickets/${TICKET}
+git reset origin/tickets/${TICKET} --hard
+git pull
+scons opt=3 -j 4
+
+printf '\nSetting up lsst-sitcom/summit_utils \n'
+cd $REPOS
+git clone https://github.com/lsst-sitcom/summit_utils.git
+cd summit_utils
+setup -j -r .
+git fetch --all
+git checkout tickets/${TICKET}
+git reset origin/tickets/${TICKET} --hard
+git pull
+scons opt=3 -j 1
+
+printf '\nSetting up lsst-sitcom/summit_extras \n'
+# printf 'Failed tests are allowed for this package'
+# Remove exit on failure
+# set +e
+cd $REPOS
+git clone https://github.com/lsst-sitcom/summit_extras.git
+cd summit_extras
+setup -j -r .
+git fetch --all
+git checkout tickets/${TICKET}
+git reset origin/tickets/${TICKET} --hard
+git pull
+scons opt=3 -j 4
+
+# set back to exit on failure
+# set -e
+
+printf '\nSetting up lsst-ts/cwfs \n'
+cd $REPOS
+git clone https://github.com/lsst-ts/cwfs.git
+cd cwfs
+git fetch --all
+setup -j -r .
+scons
+
+printf '\nSetting up lsst-ts/ts_wep \n'
+cd $REPOS
+git clone https://github.com/lsst-ts/ts_wep.git
```

Comment:
this is where we got in trouble :)
Nothing to do here... but just a reminder that we'll need to do something different for the next run.